packages/gatsby/src/redux/plugin-runner.js

```diff
@@ -8,6 +8,6 @@ emitter.on(`CREATE_PAGE`, action => {
   apiRunnerNode(
     `onCreatePage`,
     { page, traceId: action.traceId, parentSpan: action.parentSpan },
-    action.plugin.name
+    { pluginSource: action.plugin.name, activity: action.activity }
```

Comment:
Where are we using `pluginSource`?

Reply:
We changed signature of `apiRunnerNode` from https://github.com/gatsbyjs/gatsby/blob/366d915ae8db3ba96b60e8cb4deb920c08adc4e7/packages/gatsby/src/utils/api-runner-node.js#L235
so 3rd argument accept object with optional fields instead of single `pluginSource` arg.

This is later used in api runner to not call `onCreatePage` for the plugin that created given page: https://github.com/gatsbyjs/gatsby/blob/366d915ae8db3ba96b60e8cb4deb920c08adc4e7/packages/gatsby/src/utils/api-runner-node.js#L248-L255